OKTA-413125 post tuscany edits part3 and nut-fact additions (#2852)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/nodejs/integrationsteps.md

### 3: Facebook redirects the user to your Okta org

If the user Facebook login is successful, facebook routes the user to the value you enter for **Valid OAuth Redirect URIs** and **Site URL** in [Create a Facebook app in Facebook](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#_1-create-a-facebook-app-in-facebook).

> The value takes on the following format:  `https://{Okta org domain}/oauth2/v1/authorize/callback` (for example, `https://dev-12345678.okta.com/oauth2/v1/authorize/callback`).

Reviewer: ```suggestion
The value takes on the following format: `https://{Okta org domain}/oauth2/v1/authorize/callback` (for example, `https://dev-12345678.okta.com/oauth2/v1/authorize/callback`).
```

### 4: The Okta org redirects the user to your app using the sign-in redirect URI

After Facebook sends the success login request to your Okta org, the org
redirects the request to your app via the Application’s
**Sign-in redirect URIs** field.

This step handles the callback from the Widget that

> returns an `interaction_code`. This code will be redeemed in the

Reviewer: ```suggestion
returns an `interaction_code`. This code is redeemed in the
```
